### ISSUE TITLE
An Image is Worth One Word: Personalizing Text-to-Image Generation using Textual Inversion

### ISSUE THREAD

**Issue by reyllama** [reyllama/paper-reviews#182]
# TL; DR

- Invert a handful of images to a text embedding space via optimization (*Textual Inversion*) and go through conventional text-to-image.
- The inverted embedding (*pseudo word*) can be plugged into the sentence for diverse usage (ex: object, style, pose, etc)

![image](https://user-images.githubusercontent.com/46308407/185855355-b4c6389f-a860-4c02-abeb-a961fdffbb46.png)

# Motivation

- It's often not straightforward how to describe an object, style or composition in words, although natural language is one of the easiest medium.

# Formulation

![image](https://user-images.githubusercontent.com/46308407/185855564-e9aa0aee-393b-4f56-8e5d-b3f3efdd2334.png)

- Pose textual inversion as conventional optimization-based reconstruction problem, where the authors want to find some textual embedding that results in the provided images after text-to-image (by LDM).

# Evaluation

![image](https://user-images.githubusercontent.com/46308407/185856438-8b2bc360-7fbc-4c38-b630-16a1f0ee0b99.png)
![image](https://user-images.githubusercontent.com/46308407/185856754-9b6c3aef-7046-4050-88b7-2bf08e2ed348.png)
![image](https://user-images.githubusercontent.com/46308407/185856898-2966c680-54db-4fce-9ef8-e22b2745aa16.png)
![image](https://user-images.githubusercontent.com/46308407/185856934-dba69349-66b7-476d-b942-c343a90dc5ad.png)
![image](https://user-images.githubusercontent.com/46308407/185856964-2f8a022f-6afe-40db-b61e-593befe69851.png)

# Inspiration

- It's really interesting how the authors pose this problem, i.e., textual inversion.
- This work as little technical novelty in terms of the method as they employ simple optimization-based inversion widely used in GAN inversion literature.
- However, it has greater upside compared to conventional GANs as these *pseudo words* can be used in various contexts once inverted: object, style, pose, arrangement, etc.
